.github/workflows/moving-cards.yml

```diff
@@ -42,20 +43,26 @@ jobs:
       # Now handling the needed labeling
       - name: Triage labeling
         # Only if moved into triage
-        if: ${{ github.event.project_card.column_id == env.TRIAGE_COLUMN_ID  }}
+        if: ${{ github.event.project_card.column_id == env.TRIAGE_COLUMN_ID }}
         uses: andymckay/labeler@1.0.4
         with:
-          add-labels: "triage"
+          add-labels: ${{ needs.get-issue.outputs.creator == 'bitnami-bot' && 'auto-merge' || 'triage' }}
           remove-labels: "on-hold, in-progress, solved"
+      - name: Triage labeling - verify
+        # Only if moved into triage and the creator is bitnami-bot
+        if: ${{ github.event.project_card.column_id == env.TRIAGE_COLUMN_ID && needs.get-issue.outputs.creator == 'bitnami-bot' }}
+        uses: andymckay/labeler@1.0.4
+        with:
+          add-labels: verify
+          # Required to trigger CI workflow
+          repo-token: "${{ secrets.BITNAMI_BOT_TOKEN }}"
       - name: From Bitnami labeling
         # NOTE: This step has to be modified when a new column for automated PR is added
-        if: |
-          github.event.project_card.column_id == env.BITNAMI_COLUMN_ID &&
-          (needs.get-issue.outputs.creator == 'bitnami-bot')
+        if: ${{ github.event.project_card.column_id == env.BITNAMI_COLUMN_ID }}
         uses: andymckay/labeler@1.0.4
         with:
-          add-labels: "review-required"
-          remove-labels: "auto-merge"
+          add-labels: ${{ needs.get-issue.outputs.creator == 'bitnami-bot' && 'review-required' || 'bitnami' }}
```

Comment:
Careful with this. We said to add the `bitnami` label **only** when we (Bitnami team) creates the issue/PR, not when the bot does it.

Reply:
We can reach `From Bitnami` column in these cases:
* Bitnami Team member creates a PR, `triage.yml` workflow will create the card directly in in 'From Bitnami'.
* Automated PR fails, `ci-workflow.yml` will add a comment and `comments.yml` workflow will move the card to `From Bitnami`, in this case the label added will be `review-required`. NOTE: This will be changed when the new column for failed PRs is added.
* A Bitnami Team member moves the card to `From Bitnami`, I think it makes sense to have the `bitnami` label because we are using the label to track the column where the issue/PR belongs to